data/tooltips.txt

```diff
@@ -41,6 +41,12 @@ tip "afterburner heat:"
 tip "afterburner thrust:"
 	`Thrust produced by this afterburner.`
 
+tip "afterburner cooldown:"
+	`The time it takes in seconds for the afterburner to be available again after the duration ends.`
```

Comment:
It seems like the time isn't in seconds but in frames from what I can tell? Either way, it should be in frames like everything else.